Fails to start inside OpenFin

```shell
$ npm run start:openfin:local 
```

Output:
```
> npm-run-all -p openfin start:local


> reactive-trader-cloud@2.1.0 openfin /home/mozmorris/Code/Adaptive/ReactiveTraderCloud/src/client
> openfin -c public/config/openfin/local.json -l


> reactive-trader-cloud@2.1.0 start:local /home/mozmorris/Code/Adaptive/ReactiveTraderCloud/src/client
> cross-env REACT_APP_ENV=local npm run start-js

fs.js:25
'use strict';
^

ReferenceError: internalBinding is not defined
    at fs.js:25:1
    at req_ (/home/mozmorris/Code/Adaptive/ReactiveTraderCloud/src/client/node_modules/natives/index.js:137:5)
    at Object.req [as require] (/home/mozmorris/Code/Adaptive/ReactiveTraderCloud/src/client/node_modules/natives/index.js:54:10)
    at Object.<anonymous> (/home/mozmorris/Code/Adaptive/ReactiveTraderCloud/src/client/node_modules/fstream/node_modules/graceful-fs/fs.js:1:99)
    at Module._compile (internal/modules/cjs/loader.js:707:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:718:10)
    at Module.load (internal/modules/cjs/loader.js:605:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:544:12)
    at Function.Module._load (internal/modules/cjs/loader.js:536:3)
    at Module.require (internal/modules/cjs/loader.js:643:17)
npm ERR! code ELIFECYCLE
```

--- a/src/client/.gitignore
+++ b/src/client/.gitignore
@@ -2,3 +2,4 @@ build/
 .history/
 storybook-static/
 node_modules/
+OpenFin

--- a/src/client/package.json
+++ b/src/client/package.json
@@ -33,12 +33,14 @@
     "build:cloud": "cross-env REACT_APP_ENV=demo npm run build",
     "test": "react-scripts-ts test --env=jsdom",
     "analyze": "source-map-explorer build/static/js/main.*",
+    "preopenfin": "npm run sleep",
     "openfin": "openfin -c public/config/openfin/local.json -l",
     "openfin:launcher": "openfin -c public/config/openfin/launcher-local.json -l",
     "openfin-build-installer": "node ./lib/downloadInstaller.js",
     "precommit": "lint-staged",
     "storybook": "start-storybook -p 6006",
-    "storybook:build": "build-storybook"
+    "storybook:build": "build-storybook",
+    "sleep": "sleep 5"
   },
   "license": "MIT",
   "lint-staged": {

--- a/src/client/package.json
+++ b/src/client/package.json
@@ -132,7 +132,7 @@
     "lint-staged": "^7.1.1",
     "npm-run-all": "^4.1.2",
     "numeral": "^1.5.3",
-    "openfin-cli": "^1.1.6",
+    "openfin-cli": "^2.0.4",
     "prettier": "^1.12.1",
     "react-docgen-typescript-webpack-plugin": "^1.1.0",
     "react-scripts": "^1.1.5",

